docs/README.md

```diff
@@ -0,0 +1,13 @@
+# TT-UMD Docs
+
+## Requirements
+
+TODO
+
+## Build docs
+
+In order to build docs `TT_UMD_HOME` needs to be set to root of tt-umd project.
+
+After that you can run [`build_docs.sh`](build_docs.sh)
```

Comment:
This is missing from the PR, right?

Reply:
Yeah there was gitignore line to ignore everythng that starts with build. I renamed it, changed README and pushed it